pynecone/.templates/jinja/web/pages/index.js.jinja2

```diff
@@ -70,7 +71,21 @@ export default function Component() {
   {% endfor %}
 
   return (
+  <Fragment>
+      <Fragment>
+        {notConnected ? (
+          <Box sx={% raw %}{{"textAlign": "center"}}{% endraw %}>
```

Comment:
I think we can define this as a Pynecone component - and allow the user to override it as an argument to `pc.App`